### PR TITLE
create & modify tutorials/create-waggle.md

### PR DESCRIPTION
**Pull request by SammiH2022**

### CODE REVIEW

File: docs/tutorials/create-waggle.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+# Building your own Waggle Device
+
+:::
+Note: This tutorial is work in progress and is made available here for early preview.
+:::
+
+Do you ever look into the sky and wonder how will the cloud move over time? Do you ever want to know if a wildfire is possibly to start in a forest? If you would like to build, design and deploy a software that could answer your questions above, then Waggle is the right choice for you. Waggle is an edge computing wireless sensor device, say, you can define what the device will be detecting by building a software on it.
+
+If you would like to gain more understandings and share your own observation, this tutorial will give you a guide of how to register your Waggle device to the Sage network. After registering, you can publish your observation and see others data on [Sage Portal](https://portal.sagecontinuum.org/apps/explore).
```
Comment (gemblerz) [inline docs/tutorials/create-waggle.md:13]: @jswantek I want to make sure we are on the same page. We don't want to mention Sage here right?

File: docs/tutorials/create-waggle.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+
+## Potential Use Cases
+
+You might be ...
+- a professor that wants to introduce Waggle device to your students and plans to instruct students to make 20+ waggle devices
+- someone interested in Waggle and making an app but doesnt have access to production nodes
```
Comment (gemblerz) [inline docs/tutorials/create-waggle.md:20]: We can probably get rid of "but ..."
How about
```
someone interested in Waggle and making apps and use the device to test the apps
```

File: docs/tutorials/create-waggle.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+- a new user of waggle network that wants to use a new sensor (i.e. a new camera, a new bat singal detector, a new custom sensor they built)
+
+If you have your device ready, you could follow the instructions in [Access Waggle sensors](./access-waggle-sensors.md) to start defining the edge computing software on the device. If you don't, you could use a Sage Edge Computing kit to easy set up a hardware platform for development.
```
Comment (gemblerz) [inline docs/tutorials/create-waggle.md:23]: Can we link "your device" with ## Supported Platforms section?

File: docs/tutorials/create-waggle.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+An inexpensive Sage Edge Computing kit that can enable students to run existing AI@Edge computations with cameras and microphones, and then develop their own codes. The Sage Beehive will host the data and make it available to cloud-based analysis tools and workflow frameworks. This document describes the overall concept, some initial goals, and workflow.
```
Comment (gemblerz) [inline docs/tutorials/create-waggle.md:25]: We can get rid of Sage and use the term Waggle?

File: docs/tutorials/create-waggle.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+## Support Platforms
+
+The only supported platform now is Nvidia Nano, please refer to the [link](https://github.com/waggle-sensor/node-platforms/tree/main/nvidia-nano) to see the guide of setting up Nvidia Nano.
+
+In the future, we will support more [platforms](https://github.com/waggle-sensor/node-platforms) such as Raspberry Pi 4, Wild Waggle Node, etc. You can also design your own device and follow the steps below to register to the platform.
+
+## Register Your Own Device
```
Comment (gemblerz) [inline docs/tutorials/create-waggle.md:33]: Add "To Waggle"

File: docs/tutorials/create-waggle.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+In order to register your waggle device, you will need keys to authenticate your device with beehive when setting up your nano device. To get registration keys, please log in to [Sage Portal](https://portal.sagecontinuum.org/apps/explore) with your Globus credentials, after logging in, visit [My devices](https://portal.sagecontinuum.org/my-devices) page under your user account. Enter your device ID on "My Waggle Devices" page then click "Get Keys" button. It will generate a zip file for you. Check your downloads folder for the zip file, then move it to the same folder with the ansible script to proceed the execution.
```
Comment (gemblerz) [inline docs/tutorials/create-waggle.md:35]: Where is the Ansible script? can you link any resource?

File: docs/tutorials/create-waggle.md (hunk 3 above)
Comment (jswantek) [inline docs/tutorials/create-waggle.md:23]: I am going to suggest here the rest of the document. I would like to take everything you wrote and re-arrange it like the following:

```
## Getting Started

To get started in boot-strapping your Waggle Edge Computing kit you can follow the instructions for the various supported platforms on the [node-platforms](https://github.com/waggle-sensor/node-platforms) GitHub page.

> The currently only supported platform is the [NVidia nano](https://github.com/waggle-sensor/node-platforms/tree/main/nvidia-nano) but more platforms will be added in the future. In fact, if you would like to add support for other platforms go ahead and submit a [pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request) to [node-platforms](https://github.com/waggle-sensor/node-platforms).

### Registering Your Device

During the bootstrapping process you will have the option to register your device within the [web portal](https://portal.sagecontinuum.org/apps/explore). It is highly recommended to register your device, as this enables all the core [WES tools](LINK HERE) to be automatically downloaded, enabling the edge app development and run-time environment. Additionally, this enables your edge apps to publish data to the development [Beehive](LINK HERE), accessible to cloud-based analysis tools and workflow frameworks.

To register your device navigate to the [My Devices Web Portal](https://portal.sagecontinuum.org/my-devices). Enter your device ID on "My Waggle Devices" page then click "Get Keys" button. A "registration zip" file will be generated and available for download. Then follow the instructions for [your device](https://github.com/waggle-sensor/node-platforms) to load the registration keys.

> You may register as many times as you want. But note that each registration key has a short expiration time and should be used shortly after generation.
```

Then lets add a "now you a ready to develop your edge apps and/or introduce new sensors to the Waggle platform" conclusion paragraph with links to the other tutorials.

**Review by jswantek — CHANGES_REQUESTED** [review]

File: docs/tutorials/create-waggle.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
+---
```
[the remark this answers is not in the document]
Comment (SammiH2022) [reply docs/tutorials/create-waggle.md:1]: Does this replace the `To integrate your sensor device into Waggle, head over to the [Contact Us](../contact-us.md) page` in `./access-waggle-sensors.md`?

File: docs/tutorials/create-waggle.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+sidebar_position: 4
+---
+
+# Building your own Waggle Device
+
+:::
+Note: This tutorial is work in progress and is made available here for early preview.
+:::
+
+Do you ever look into the sky and wonder how will the cloud move over time? Do you ever want to know if a wildfire is possibly to start in a forest? If you would like to build, design and deploy a software that could answer your questions above, then Waggle is the right choice for you. Waggle is an edge computing wireless sensor device, say, you can define what the device will be detecting by building a software on it.
```
[the remark this answers is not in the document]
Comment (SammiH2022) [reply docs/tutorials/create-waggle.md:11]: Replaced the whole paragraph with [this comment](https://github.com/waggle-sensor/waggle-docs/pull/12#discussion_r979026746)

File: docs/tutorials/create-waggle.md (hunk 5 above)
Comment (SammiH2022) [reply docs/tutorials/create-waggle.md:33]: replaced with [this comment](https://github.com/waggle-sensor/waggle-docs/pull/12#discussion_r979064070)

File: docs/tutorials/create-waggle.md (hunk 4 above)
Comment (SammiH2022) [reply docs/tutorials/create-waggle.md:25]: yes, I deleted all `sage` in the page

File: docs/tutorials/create-waggle.md (hunk 3 above)
Comment (SammiH2022) [reply docs/tutorials/create-waggle.md:23]: replaced with [this comment](https://github.com/waggle-sensor/waggle-docs/pull/12#discussion_r979064070)

File: docs/tutorials/create-waggle.md (hunk 2 above)
Comment (SammiH2022) [reply docs/tutorials/create-waggle.md:20]: replace with [this comment](https://github.com/waggle-sensor/waggle-docs/pull/12#discussion_r979064070)

File: docs/tutorials/create-waggle.md (hunk 6 above)
Comment (SammiH2022) [reply docs/tutorials/create-waggle.md:35]: this paragraph is replaced with https://github.com/waggle-sensor/waggle-docs/pull/12#discussion_r979064070

File: docs/about/overview.md [hunk 9]
```diff
@@ -57,8 +57,9 @@ The entry-point into learning about your interaction with Waggle can be best dir
 - Want to get a text message when your edge application does something cool?
 
 [How do I build my own Waggle device?](../tutorials/create-waggle.md)
-- Want to set up your own device to build up an edge app on it?
-- Want to teach AI to a classroom of students with an instruction manual?
+
```
Comment (jswantek) [inline docs/about/overview.md:60]: remove the extra space here

File: docs/tutorials/create-waggle.md [hunk 10]
```diff
@@ -1,37 +1,28 @@
...
+sidebar_position: 5
 ---
 
-# Building your own Waggle Device
+# Building your own Waggle device
 
-:::
-Note: This tutorial is work in progress and is made available here for early preview.
-:::
+Are you a professor that wants to use inexpensive Waggle devices to teach students interested in AI? Are you someone interested in developing a new [edge app](./edge-apps/1-intro-to-edge-apps.md) using a local development platform? Are you a Waggle user interested in using a new sensor (i.e. a new camera, a bat signal detector, a custom sensor they built)? If you would like to build, design and deploy software that could answer your questions above, then Waggle is the right choice for you.
```
Comment (jswantek) [inline docs/tutorials/create-waggle.md:7]: lets switch ‘inexpensive’ with ‘affordable’

File: docs/tutorials/create-waggle.md [hunk 11]
```diff
@@ -1,37 +1,28 @@
...
 
-Do you ever look into the sky and wonder how will the cloud move over time? Do you ever want to know if a wildfire is possibly to start in a forest? If you would like to build, design and deploy a software that could answer your questions above, then Waggle is the right choice for you. Waggle is an edge computing wireless sensor device, say, you can define what the device will be detecting by building a software on it.
+This tutorial will guide you in preparing your own Waggle device and (optionally) registering it to upload data to a shared [development Beehive](../about/architecture.md#beehive). This Waggle device is a fully unlocked development platform running the same [WES infrastructure](../about/architecture.md#waggle-edge-stack-wes) that runs in production Waggle edge devices (ex. the [Wild Waggle Node](../about/architecture.md#wild-waggle-node)). This is an ideal platform for user's interested in developing a new [edge app](./edge-apps/1-intro-to-edge-apps.md) and/or experimenting with a [new sensor](./access-waggle-sensors.md#bring-your-own-sensor-to-waggle).
```
Comment (jswantek) [inline docs/tutorials/create-waggle.md:9]: `user's` -> `users`

File: docs/tutorials/create-waggle.md [hunk 12]
```diff
@@ -1,37 +1,28 @@
...
 
-In the future, we will support more [platforms](https://github.com/waggle-sensor/node-platforms) such as Raspberry Pi 4, Wild Waggle Node, etc. You can also design your own device and follow the steps below to register to the platform.
 
-## Register Your Own Device
-
-In order to register your waggle device, you will need keys to authenticate your device with beehive when setting up your nano device. To get registration keys, please log in to [Sage Portal](https://portal.sagecontinuum.org/apps/explore) with your Globus credentials, after logging in, visit [My devices](https://portal.sagecontinuum.org/my-devices) page under your user account. Enter your device ID on "My Waggle Devices" page then click "Get Keys" button. It will generate a zip file for you. Check your downloads folder for the zip file, then move it to the same folder with the ansible script to proceed the execution.
-
-![Figure 1: Sage Portal - My devices](./images/sage-my-devices.png)
\ No newline at end of file
+![Figure 1: Registering my devices](./images/sage-my-devices.png)
```
Comment (jswantek) [inline docs/tutorials/create-waggle.md:28]: Move image above the note (`> You may register as many`) above to git the image some context

**Review by jswantek — CHANGES_REQUESTED** [review]

File: docs/tutorials/create-waggle.md [hunk 13]
```diff
@@ -1,37 +1,28 @@
...
-- someone interested in Waggle and making an app but doesnt have access to production nodes
-- a new user of waggle network that wants to use a new sensor (i.e. a new camera, a new bat singal detector, a new custom sensor they built)
+### Registering your Waggle device
 
-If you have your device ready, you could follow the instructions in [Access Waggle sensors](./access-waggle-sensors.md) to start defining the edge computing software on the device. If you don't, you could use a Sage Edge Computing kit to easy set up a hardware platform for development.
+During the bootstrapping process you will have the option to register your device within the [web portal](https://portal.sagecontinuum.org/apps/explore). It is highly recommended to register your device, as this enables all the core [WES tools](LINK HERE) to be automatically downloaded, enabling the edge app development and run-time environment. Additionally, this enables your edge apps to publish data to the development [Beehive](LINK HERE), accessible to cloud-based analysis tools and workflow frameworks.
```
[the remark this answers is not in the document]
Comment (SammiH2022) [reply docs/tutorials/create-waggle.md:19]: @jswantek Does it make sense to put `https://docs.waggle-edge.ai/docs/about/architecture#waggle-edge-stack-wes` for WES tools and `https://docs.waggle-edge.ai/docs/about/architecture#beehive` for Beehive?

File: docs/tutorials/create-waggle.md [hunk 14]
```diff
@@ -1,37 +1,28 @@
...
 
-An inexpensive Sage Edge Computing kit that can enable students to run existing AI@Edge computations with cameras and microphones, and then develop their own codes. The Sage Beehive will host the data and make it available to cloud-based analysis tools and workflow frameworks. This document describes the overall concept, some initial goals, and workflow.
+To register your device navigate to the [My Devices Web Portal](https://portal.sagecontinuum.org/my-devices). Enter your device ID on "My Waggle Devices" page then click "Get Keys" button. A "registration zip" file will be generated and available for download. Then follow the instructions for [your device](https://github.com/waggle-sensor/node-platforms) to load the registration keys.
```
[the remark this answers is not in the document]
Comment (SammiH2022) [reply docs/tutorials/create-waggle.md:21]: I keep the `My Devices Web Portal` and delete the other one.

**Review by jswantek — APPROVED** [review]

**Pull request merged**